packages/nextjs/src/utils/handlers.ts

```diff
@@ -0,0 +1,23 @@
+import { captureException, flush, Handlers, withScope } from '@sentry/node';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const { parseRequest } = Handlers;
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const withSentry = (handler: (req: NextApiRequest, res: NextApiResponse) => Promise<void>) => {
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+  return async (req: NextApiRequest, res: NextApiResponse) => {
+    try {
+      // TODO: Start Transaction
```

Comment:
Should we also extract data from the request? https://github.com/getsentry/sentry-javascript/pull/3463/files#diff-1d2c03eeaa78dd1674379a1628e39b2026cb775d32aea72a366702db9aba1325R10

Reply:
https://github.com/getsentry/sentry-javascript/pull/3462/files#diff-1d2c03eeaa78dd1674379a1628e39b2026cb775d32aea72a366702db9aba1325R16